package/SUSEConnect.changes

```diff
@@ -1,4 +1,10 @@
--------------------------------------------------------------------
+--------------------------------------------------------------------
+Mon Jul 4 10:00:00 UTC 2022 - Natnael Getahun <ngetahun@suse.com>
+
+- Update to 0.3.35
+- Remove automatic enablement of sync timer. (bsc#1200641)
```

Comment:
I'd suggest to word this: "Rely on system-wide defaults for enabling the keepalive timer by systemd-presets-branding-SLE". 
Technically it's correct that we're dropping the automatic enablement of the timer here, but for the user it's more informative to share what's going on.